lia-ui/src/main/java/com/lithium/community/android/ui/components/adapters/LiCreateMessageAdapter.java

```diff
@@ -52,9 +53,21 @@
     private LiCreateMessageFragment liCreateMessageFragment;
     private Activity activity;
     private String htmlTemplate;
+    private String currentMessage;
+    private String currentTitle;
+
+    public void setCurrentMessage(String currentMessage) {
+        this.currentMessage = currentMessage;
+        notifyDataSetChanged();
+    }
+
+    public void setCurrentTitle(String title){
+        this.currentTitle = title;
+        notifyDataSetChanged();
+    }
```

Comment:
setters before the constructor. Can we move them below?

Reply:
sure